core/os/os.h

```diff
@@ -255,6 +255,7 @@ class OS {
 	virtual String get_executable_path() const;
 	virtual Error execute(const String &p_path, const List<String> &p_arguments, bool p_blocking, ProcessID *r_child_id = NULL, String *r_pipe = NULL, int *r_exitcode = NULL, bool read_stderr = false) = 0;
 	virtual Error kill(const ProcessID &p_pid) = 0;
+	virtual Error _stop_or_kill(const ProcessID &p_pid, const int stop_max_wait_msec);
```

Comment:
If it's public it should not start with `_` IMO. Also note that parameters should start with `p_`, so it should be `p_stop_max_wait_msec`.

Reply:
Updated to ````p_max_wait_msec````; also merged ````_stop_or_kill```` with ````kill````